### ISSUE TITLE
Repurposing Masks into Bounding Boxes.

### ISSUE THREAD

**Issue by innat** [keras-team/keras-cv#173]
**Convert the Segmentation masks into Object localization annotations for different tasks, e.g. Instance and Panoptic segmentation.**

Reference: [torchvision.ops.masks_to_boxes](https://pytorch.org/vision/stable/generated/torchvision.ops.masks_to_boxes.html#torchvision.ops.masks_to_boxes)
Details and Code: [Mask-to-BoundingBox](https://pytorch.org/vision/stable/auto_examples/plot_repurposing_annotations.html#sphx-glr-auto-examples-plot-repurposing-annotations-py)

Demo.

![sphx_glr_plot_repurposing_annotations_001](https://user-images.githubusercontent.com/17668390/158814358-12539a76-cd20-4fa9-94a6-2d2278c4e176.png) ![sphx_glr_plot_repurposing_annotations_002](https://user-images.githubusercontent.com/17668390/158814387-157ebb46-4d68-420a-bf92-817fea627c46.png)

**Comment by kartik4949** [comment]
Well, 
I had made an utility already in one of my project, if needed I can contribute in above

usecases

converting mask dataset to bbox

**Comment by innat** [comment]
That would be great. I think it will fall under utility function.

**Comment by LukeWood** [comment]
yeah I think this is a reasonable utility function to have

**Comment by kartik4949** [comment]
Let me contribute for the same.
